trademarks.md

```diff
@@ -0,0 +1,144 @@
+- Use of the word "Jupyter" in company names -- Allowed only by prior written permission from Jupyter.
+- Use of the word "Jupyter" when redistributing Jupyter software as part of a freely distributed application -- Allowed. If the standard version of the Jupyter software is modified, this should be clearly indicated. For commercial distributions, contact Jupyter for permission if your use is not covered by the nominative use rules described in the section "Uses that Never Require Approval" above.
+- Use of the word "Jupyter" in the names of user groups and conferences that are free to join or attend (Ex., "Jupyter Meetup") -- Allowed if referring to the Jupyter software. Other uses require permission.
+- Use of the word "Jupyter" in the name of books or publications like "Jupyter Journal" and "Jupyter Cookbook" -- Allowed if referring to the Jupyter software.
+- Use of the word "Jupyter" on websites, brochures, documentation, and product packaging -- Allowed if referring to the Jupyter software. Please follow the rules above about the use of the TM symbol.
+- Use of the word "Jupyter" in advertisements -- Allowed in most cases by the nominative use rules described in the section "Uses that Never Require Approval" above. Other uses in ads only with prior permission.
+- Use of the word "Jupyter" in email and informally -- Allowed without the TM symbol.
+- Use of the word "Jupyter" in academic papers, theses, and books -- Allowed without the TM symbol. Books should include the symbol.
+- Use of the word "Jupyter" in another trademark -- Not allowed without prior written permission from Jupyter, except as described above.
+- Use of the word "Jupyter" in a domain name -- Allowed in subdomains and url-paths,
+  such as "jupyter.example.com" and "example.com/jupyter".
+  Not allowed in base domains, such as "jupytercloud.com" or "hostedjupyter.horse".
+
+
+### Unaltered Logos
+
+- Use of unaltered Jupyter-provided logos on T-shirts, mugs, etc. -- Again, non-commercial uses to promote the Jupyter software are allowed.
+    Commercial uses (which includes any use where you sell these items for money) require permission from Project Jupyter.
+    Please reproduce our logos with the right colors and fonts; if you need help, let us know.
+    See the [brand guidelines][].
+- Use of unaltered Jupyter-provided logos on websites, brochures, and product packaging. 
+    Non-commercial uses to promote the Jupyter software are allowed, as are all nominative uses as described in the section "Uses that Never Require Approval".
+    Any other commercial uses require prior written permission from Project Jupyter.
+    See the [brand guidelines][].
+
+
+[brand guidelines]: https://github.com/jupyter/design/tree/master/brandguide
+[Jupyter Trademarks Committee]: mailto:jupyter-trademarks@googlegroups.com
```

Comment:
Probably at the end, though it could also be at the beginning, there should be an acknowledgment to the original PSF doc.